src/utils.h

```diff
@@ -157,23 +157,23 @@ namespace pyne {
     ~FileNotFound () throw () {};
 
     /// constructor with the filename \a fname.
-    FileNotFound(std::string fname)
+    FileNotFound(std::string fname) : FNF_message("File not found: ")
     {
       filename = fname;
     };
 
     /// Creates a helpful error message.
     virtual const char* what() const throw()
     {
-      std::string FNFstr ("File not found: ");
       if (!filename.empty())
-        FNFstr += filename;
+        FNF_message += filename;
```

Comment:
I couldn't get this to compile. I think because the method is `const`? 

I'd say we could remove `'const` but the `std::exception` method it inherits from is also `const`. As we probably want to match that interface as closely as possible, I'd recommend adding `mutable` in front of the FNF_message declaration.

Reply:
Or maybe I'm missing something here. LMK!

Reply:
Yeah - I'm using CI for testing :blush:

Reply:
Fixed this by setting the full message in the constructor